index.bs

```diff
@@ -174,11 +174,9 @@ interface IntersectionObserver {
 		3. Set |this|.{{IntersectionObserver/root}} to
 			|options|.{{IntersectionObserverInit/root}}
 		4. Attempt to <a>parse a root margin</a>
-			from |options|.{{IntersectionObserverInit/rootMargin}}.
-			If a list is returned,
+			from |options.rootMargin|.  If a list is returned,
```

Comment:
This line doesn't seem to translate correctly into _options.rootMargin_ in the output html.